adapters/AdColony/AdColonyAdapter/GADMAdapterAdColonyHelper.m

```diff
@@ -101,6 +101,13 @@ + (nullable AdColonyAdOptions *)getAdOptionsFromAdConfig:
     // Popups only apply to rewarded ads.
     options = [self getAdOptionsFromExtras:extras];
   }
+    
+  if (adConfig.bidResponse) {
+    if (options == nil) {
+      options = [AdColonyAdOptions new];
+    }
+    [options setOption:@"adm" withStringValue:adConfig.bidResponse];
```

Comment:
@mahesh-kokate Is this a special key for AdColony that says to render an ad with this bid response? Should we make it a constant?

Reply:
Hi @ericleich, yes we can make a constant of it.

Reply:
Done, I have created constant for it. Thanks.

Reply:
Hi @ericleich, We have released AdColony [SDK v4.3.0](https://github.com/AdColony/AdColony-iOS-SDK) last week, so I have updated same PR for it. Please let me know if you have any questions.